diff --git a/makefile.gen b/makefile.gen
index b9d7877..07d96af 100644
--- a/makefile.gen
+++ b/makefile.gen
@@ -24,6 +24,8 @@ BINTOS= $(BIN)/bintos
 RESCOMP= $(BIN)/rescomp
 MKDIR= $(BIN)/mkdir
 
+include $(GENDEV)/sgdk/mkfiles/makefile.vars
+
 SRC_C= $(wildcard *.c)
 SRC_C+= $(wildcard $(SRC)/*.c)
 SRC_S= $(wildcard *.s)
diff --git a/res/libres.h b/res/libres.h
index 1f6155d..f31e5b5 100644
--- a/res/libres.h
+++ b/res/libres.h
@@ -1,9 +1,9 @@
-#ifndef _D__APPS_SGDK_RES_LIBRES_H_
-#define _D__APPS_SGDK_RES_LIBRES_H_
+#ifndef _RES_LIBRES_H_
+#define _RES_LIBRES_H_
 
 extern const Bitmap logo_lib;
 extern const TileSet font_lib;
 extern const Palette font_pal_lib;
 extern const u8 stop_xgm[172];
 
-#endif // _D__APPS_SGDK_RES_LIBRES_H_
+#endif // _RES_LIBRES_H_
index 451564b..37b72b7 100644
--- a/sample/hs_effect/src/boot/rom_head.c
+++ b/sample/hs_effect/src/boot/rom_head.c
@@ -1,6 +1,6 @@
 #include "types.h"
 
-
+__attribute__((externally_visible))
 const struct
 {
     char console[16];               /* Console Name (16) */
index 451564b..37b72b7 100644
--- a/sample/joytest/src/boot/rom_head.c
+++ b/sample/joytest/src/boot/rom_head.c
@@ -1,6 +1,6 @@
 #include "types.h"
 
-
+__attribute__((externally_visible))
 const struct
 {
     char console[16];               /* Console Name (16) */
index 24d748b..ab8eb69 100644
--- a/sample/sound/res/resources.res
+++ b/sample/sound/res/resources.res
@@ -7,7 +7,7 @@ XGM midnight "midnight.vgm"
 VGM roadrash "roadrash.vgm"
 VGM sonic1 "sonic1.vgm"
 VGM bapcm_vgm "bad_apple_pcm.vgm"
-TFM music_tfd "music_tfd.tfd" 1
+#TFM music_tfd "music_tfd.tfd" 1
 BIN music_mvs "music_mvs.mvs" 256
 
 WAV india_8k "india_8k.wav" 0 8000
diff --git a/sample/sound/src/main.c b/sample/sound/src/main.c
index 117e891..498160b 100644
--- a/sample/sound/src/main.c
+++ b/sample/sound/src/main.c
@@ -568,19 +568,6 @@ static void joyEvent(u16 joy, u16 changed, u16 state)
             break;
         }
 
-        case Z80_DRIVER_TFM:
-        {
-            if (changed & state & BUTTON_A)
-            {
-                SND_startPlay_TFM(music_tfd);
-            }
-            if (changed & state & BUTTON_START)
-            {
-                SND_stopPlay_TFM();
-            }
-            break;
-        }
-
         case Z80_DRIVER_VGM:
         {
             if (changed & state & BUTTON_X)
diff --git a/src/z80_drv1.h b/src/z80_drv1.h
index 06adade..a2da7b8 100644
--- a/src/z80_drv1.h
+++ b/src/z80_drv1.h
@@ -1,6 +1,6 @@
-#ifndef _Z80_DRV1_H_
-#define _Z80_DRV1_H_
-
-extern const u8 z80_drv1[0xB2B];
-
-#endif // _Z80_DRV1_H_
+#ifndef _Z80_DRV1_H_
+#define _Z80_DRV1_H_
+
+extern const u8 z80_drv1[0xB2B];
+
+#endif // _Z80_DRV1_H_
diff --git a/src/z80_drv2.h b/src/z80_drv2.h
index 3598de1..7ef3aba 100644
--- a/src/z80_drv2.h
+++ b/src/z80_drv2.h
@@ -1,6 +1,6 @@
-#ifndef _Z80_DRV2_H_
-#define _Z80_DRV2_H_
-
-extern const u8 z80_drv2[0xE00];
-
-#endif // _Z80_DRV2_H_
+#ifndef _Z80_DRV2_H_
+#define _Z80_DRV2_H_
+
+extern const u8 z80_drv2[0xE00];
+
+#endif // _Z80_DRV2_H_
diff --git a/src/z80_drv3.h b/src/z80_drv3.h
index 8b35866..94c8000 100644
--- a/src/z80_drv3.h
+++ b/src/z80_drv3.h
@@ -1,6 +1,6 @@
-#ifndef _Z80_DRV3_H_
-#define _Z80_DRV3_H_
-
-extern const u8 z80_drv3[0xA69];
-
-#endif // _Z80_DRV3_H_
+#ifndef _Z80_DRV3_H_
+#define _Z80_DRV3_H_
+
+extern const u8 z80_drv3[0xA69];
+
+#endif // _Z80_DRV3_H_
diff --git a/src/z80_mvs.h b/src/z80_mvs.h
index 031a90f..661cbb0 100644
--- a/src/z80_mvs.h
+++ b/src/z80_mvs.h
@@ -1,6 +1,6 @@
-#ifndef _Z80_MVS_H_
-#define _Z80_MVS_H_
-
-extern const u8 z80_mvs[0xA5B];
-
-#endif // _Z80_MVS_H_
+#ifndef _Z80_MVS_H_
+#define _Z80_MVS_H_
+
+extern const u8 z80_mvs[0xA5B];
+
+#endif // _Z80_MVS_H_
diff --git a/src/z80_tfm.h b/src/z80_tfm.h
index 31e35a0..fe6fed3 100644
--- a/src/z80_tfm.h
+++ b/src/z80_tfm.h
@@ -1,6 +1,6 @@
-#ifndef _Z80_TFM_H_
-#define _Z80_TFM_H_
-
-extern const u8 z80_tfm[0x69D];
-
-#endif // _Z80_TFM_H_
+#ifndef _Z80_TFM_H_
+#define _Z80_TFM_H_
+
+extern const u8 z80_tfm[0x69D];
+
+#endif // _Z80_TFM_H_
diff --git a/src/z80_xgm.h b/src/z80_xgm.h
index 167307c..49364d2 100644
--- a/src/z80_xgm.h
+++ b/src/z80_xgm.h
@@ -1,6 +1,6 @@
-#ifndef _Z80_XGM_H_
-#define _Z80_XGM_H_
-
-extern const u8 z80_xgm[0x1700];
-
-#endif // _Z80_XGM_H_
+#ifndef _Z80_XGM_H_
+#define _Z80_XGM_H_
+
+extern const u8 z80_xgm[0x1700];
+
+#endif // _Z80_XGM_H_
diff --git a/tools/rescomp/inc/tools.h b/tools/rescomp/inc/tools.h
index 320fd16..20ef97e 100644
--- a/tools/rescomp/inc/tools.h
+++ b/tools/rescomp/inc/tools.h
@@ -1,6 +1,11 @@
 #ifndef _TOOLS_H_
 #define _TOOLS_H_
 
+#if __linux
+#define stricmp strcasecmp
+#define strnicmp strncasecmp
+#define strupr toupper
+#endif
 
 #define FALSE           0
 #define TRUE            1
diff --git a/tools/rescomp/src/tools.c b/tools/rescomp/src/tools.c
index 8160f36..c94dbcf 100644
--- a/tools/rescomp/src/tools.c
+++ b/tools/rescomp/src/tools.c
@@ -797,7 +797,7 @@ static int appack(char* fin, char* fout)
     strcat(cmd, "\" \"");
     strcat(cmd, fout);
     strcat(cmd, "\"");
-    strcat(cmd, " -s");
+    //strcat(cmd, " -s");
 
     printf("Executing %s\n", cmd);
 
