diff --git a/makefile.gen b/makefile.gen
index 709b276..c637db1 100644
--- a/makefile.gen
+++ b/makefile.gen
@@ -32,6 +32,8 @@ BINTOS := $(BIN)/bintos
 RESCOMP := $(JAVA) -jar $(BIN)/rescomp.jar
 MKDIR := $(BIN)/mkdir
 
+include $(GENDEV)/sgdk/mkfiles/makefile.vars
+
 SRC_C = $(wildcard *.c)
 SRC_C += $(wildcard $(SRC)/*.c)
 SRC_C += $(wildcard $(SRC)/*/*.c)
diff --git a/makelib.gen b/makelib.gen
index 97318ff..9d333a3 100644
--- a/makelib.gen
+++ b/makelib.gen
@@ -26,6 +26,8 @@ BINTOS := $(BIN)/bintos
 RESCOMP := $(JAVA) -jar $(BIN)/rescomp.jar
 MKDIR := $(BIN)/mkdir
 
+include ../files/makefile.vars
+
 SRC_LIB_C := $(wildcard $(SRC_LIB)/*.c)
 SRC_LIB_S := $(wildcard $(SRC_LIB)/*.s)
 SRC_LIB_S80 := $(wildcard $(SRC_LIB)/*.s80)
@@ -101,11 +103,11 @@ cleanAsm: cleanasm
 
 
 $(LIB)/libmd.a: cmd_
-	$(AR) rs $(LIB)/libmd.a --plugin=liblto_plugin-0.dll @cmd_
+	$(AR) rs $(LIB)/libmd.a @cmd_
 	$(RM) cmd_
 
 $(LIB)/libmd_debug.a: cmd_
-	$(AR) rs $(LIB)/libmd_debug.a --plugin=liblto_plugin-0.dll @cmd_
+	$(AR) rs $(LIB)/libmd_debug.a @cmd_
 	$(RM) cmd_
 
 cmd_ : $(OBJ_LIB)
