Index: inc/tools.h
===================================================================
--- inc/tools.h	(revision 206)
+++ inc/tools.h	(working copy)
@@ -1,6 +1,11 @@
 #ifndef _TOOLS_H_
 #define _TOOLS_H_
 
+#if __linux
+#define stricmp strcasecmp
+#define strnicmp strncasecmp
+#define strupr toupper
+#endif
 
 #define FALSE           0
 #define TRUE            1
Index: src/tools.c
===================================================================
--- src/tools.c	(revision 206)
+++ src/tools.c	(working copy)
@@ -798,7 +798,7 @@
     strcat(cmd, "\" \"");
     strcat(cmd, fout);
     strcat(cmd, "\"");
-    strcat(cmd, " -s");
+    //strcat(cmd, " -s");
 
     printf("Executing %s\n", cmd);
 
